api.md

```diff
@@ -778,11 +779,7 @@ partition across
 
 #### Returns
 
-| Column               | Description                              |
-|----------------------|------------------------------------------|
-| `hypertable_id`      | Hypertable id of the modified hypertable |
-| `node_hypertable_id` | Hypertable id on the remote data node    |
-| `node_name`          | Name of the attached data node     |
+The number of data nodes detached.
```

Comment:
Hmmm, we're only detaching one specifc data node so this seems odd. I suggest writing something like:

```
The number of hypertables this data node was detached from.
```

Reply:
Done.